src/code/z_message_nes.c

```diff
@@ -288,8 +290,1589 @@ void Message_LoadOwlWarpTextNES(PlayState* play, s32* offset, f32* arg2, s16* de
+                if ((msgCtx->msgMode >= MSGMODE_NEW_CYCLE_0) && (msgCtx->msgMode <= MSGMODE_OWL_SAVE_2) &&
+                    (character == 2)) {
+                    msgCtx->textDrawPos = msgCtx->decodedTextLen;
+                    if (msgCtx->unk120D6) {
+                        msgCtx->unk120D4 += 25;
+                        if (msgCtx->unk120D4 >= 100) {
+                            msgCtx->unk120D6 = false;
+                        }
+                    } else {
+                        msgCtx->unk120D4 -= 25;
+                        if (msgCtx->unk120D4 <= -100) {
+                            msgCtx->unk120D6 = true;
+                        }
+                    }
+
+                    if (D_801D07DC[(s16)(character - 1)][0] + msgCtx->unk120D4 < 0) {
+                        msgCtx->textColorR = 0;
+                    } else {
+                        msgCtx->textColorR = D_801D07DC[(s16)(character - 1)][0] + msgCtx->unk120D4;
+                    }
+
+                    if (D_801D07DC[(s16)(character - 1)][1] + msgCtx->unk120D4 >= 255) {
+                        msgCtx->textColorG = D_801D07DC[(s16)(character - 1)][1];
+                    } else {
+                        msgCtx->textColorG = D_801D07DC[(s16)(character - 1)][1] + msgCtx->unk120D4;
+                    }
+
+                    if (D_801D07DC[(s16)(character - 1)][2] + msgCtx->unk120D4 < 0) {
+                        msgCtx->textColorB = 0;
+                    } else {
+                        msgCtx->textColorB = D_801D07DC[(s16)(character - 1)][2] + msgCtx->unk120D4;
+                    }
+
+                } else if (play->pauseCtx.bombersNotebookOpen) {
+                    msgCtx->textColorR = D_801D089C[(s16)(character - 1)][0];
+                    msgCtx->textColorG = D_801D089C[(s16)(character - 1)][1];
+                    msgCtx->textColorB = D_801D089C[(s16)(character - 1)][2];
+                } else if (msgCtx->textBoxType == TEXTBOX_TYPE_1) {
+                    msgCtx->textColorR = D_801D07DC[(s16)(character - 1)][0];
+                    msgCtx->textColorG = D_801D07DC[(s16)(character - 1)][1];
+                    msgCtx->textColorB = D_801D07DC[(s16)(character - 1)][2];
+                } else if (msgCtx->textBoxType == TEXTBOX_TYPE_D) {
+                    msgCtx->textColorR = D_801D086C[(s16)(character - 1)][0];
+                    msgCtx->textColorG = D_801D086C[(s16)(character - 1)][1];
+                    msgCtx->textColorB = D_801D086C[(s16)(character - 1)][2];
+                } else {
+                    msgCtx->textColorR = D_801D080C[(s16)(character - 1)][0];
+                    msgCtx->textColorG = D_801D080C[(s16)(character - 1)][1];
+                    msgCtx->textColorB = D_801D080C[(s16)(character - 1)][2];
+                }
+
+                if ((i + 1) == msgCtx->textDrawPos) {
+                    msgCtx->textDrawPos++;
+                }
+                break;
+
+            case 0x20: // ` `
```

Comment:
If this char is known are the others known? And can be added to the cases?

Reply:
I'd refer to this comment:
https://github.com/zeldaret/mm/pull/1376#discussion_r1325322075

All the cases aren't known, and there's 3 different set of values between the 3 message system. The characters probably won't be resolved until text extraction occurs